src/lib/core/say.h

```diff
@@ -326,8 +326,7 @@ CFORMAT(printf, 5, 6) extern sayfunc_t _say;
  * \sa enum say_level
  */
 #define say_file_line(level, file, line, error, format, ...) ({ \
-	if (say_log_level_is_enabled(level)) \
-		_say(level, file, line, error, format, ##__VA_ARGS__); })
```

Comment:
The log level check was here for a reason, I think: we don't want to evaluate the format if the message isn't going to come through. Evaluating all debug-level messages unconditionally may be expensive.